src/event/cloudevent.ts

```diff
@@ -167,6 +168,17 @@ export class CloudEvent implements CloudEventV1, CloudEventV03 {
     }
   }
 
+  /**
+   * Emit this CloudEvent through the application
+   *
+   * @param {boolean} ensureDelivery fail the promise if one listener fail
+   * @return {Promise<CloudEvent>} this
+   */
+  public async emit(ensureDelivery = false): Promise<this> {
```

Comment:
I wonder if this shouldn't default to `true`. Doesn't it seem that it would typically be bad practice to "fire and forget"?
```suggestion
  public async emit(ensureDelivery = true): Promise<this> {
```

Reply:
I put false by default as this is the behavior of EventEmitter, I suppose both are defendable, if you have several subscribers maybe only is critical.
I suppose in my future implementation I would go for `true` with the main subscriber and optional one